.github/workflows/ci-publish-base-image.yml

```diff
@@ -0,0 +1,37 @@
+name: Publish Base CI Image To Dockerhub
```

Comment:
I built this first to build the image.   Once we have automated environment support I could add this to the main ci-test.yml and conditionally trigger it on lands to main/master.   Kinda want to get down to one workflow file with condition triggers for jobs.   That way we can share state with caches easily.